src/Electrical/Digital/logic_vectors.jl

```diff
@@ -0,0 +1,57 @@
+import Base: size, axes, getindex, setindex!
+
+const LogicOrNumber = Union{Logic, Number}
+
+struct StdULogicVector{N} <: AbstractArray{Logic, N}
+    logic::Array{Logic}
+    level::Array{Int}
```

Comment:
Why do we need to store `level` at all if it can be trivially computed from `logic`?

Reply:
Should we have a logic table so that it takes `Logic` indices, then?

Reply:
This point is still not addressed. We need `LogicTable{N} <: AbstractArray{Logic, N}` such that we can index into it using `Logic` indices, so we don't even need to store `level`.

Reply:
This seemed like the simplest approach to index the values.
I see the point in a `LogicTable` type. Will implement that.